media-tv/mythtv/mythtv-31.0.ebuild

```diff
@@ -0,0 +1,431 @@
+	elif use perl; then
+		myconf+=(--without-bindings=python)
+		myconf+=(--with-bindings=perl)
+	elif use python; then
+		myconf+=(--without-bindings=perl)
+		myconf+=(--with-bindings=python)
+	else
+		myconf+=(--without-bindings=perl,python)
+	fi
+	use python && myconf+=(--python="${EPYTHON}")
+	myconf+=($(use_enable java bdjava))
+
+	# External codec library options (used for mythffmpeg and streaming transcode)
+	# lame is required for some broadcasts for silence detection of commercials
+	# default enable in IUSE with +lame
+	myconf+=(
+		$(use_enable lame libmp3lame)
+		$(use_enable xvid libxvid)
+		$(use_enable x264 libx264)
+		$(use_enable x265 libx265)
+		$(use_enable vpx libvpx)
+	)
+
+	# Clean up DSO load times and other compiler bits
+	myconf+=(--enable-symbol-visibility)
+	myconf+=(--enable-pic)
+
+	if tc-is-cross-compiler ; then
+		myconf+=(--enable-cross-compile --arch=$(tc-arch-kernel))
+		myconf+=(--cross-prefix="${CHOST}"-)
+	fi
+
+	myconf+=($(use_enable bluray libbluray_external))
+
+	# econf sets these options that are not handled by configure:
+	# --build --host --infodir --localstatedir --sysconfdir
+
+	einfo "Running ./configure ${myconf[@]} - THIS MAY TAKE A WHILE."
+	./configure \
+		--prefix="${EPREFIX}/usr" \
+		--cc="$(tc-getCC)" \
+		--cxx="$(tc-getCXX)" \
+		--ar="$(tc-getAR)" \
+		--optflags="${CFLAGS}" \
+		--extra-cflags="${CFLAGS}" \
+		--extra-cxxflags="${CXXFLAGS}" \
+		--extra-ldflags="${LDFLAGS}" \
+		--qmake=$(qt5_get_bindir)/qmake \
+		"${myconf[@]}"
+}
+
+src_install() {
+	emake STRIP="true" INSTALL_ROOT="${D}" install
+	use python && python_optimize  # does all packages by default
+	dodoc AUTHORS UPGRADING README
+	readme.gentoo_create_doc
+
+	insinto /usr/share/mythtv/database
+	doins database/*
+
+	newinitd "${FILESDIR}"/mythbackend.init-r3 mythbackend
+	newconfd "${FILESDIR}"/mythbackend.conf-r1 mythbackend
+	if use systemd; then
+		systemd_newunit "${FILESDIR}"/mythbackend.service-28 mythbackend.service
+	fi
+
+	dodoc keys.txt
+
+	keepdir /etc/mythtv
+	fowners -R mythtv /etc/mythtv
+	keepdir /var/log/mythtv
+	fowners -R mythtv /var/log/mythtv
+
+	insinto /etc/logrotate.d
+	newins "${FILESDIR}"/mythtv.logrotate.d-r4 mythtv
+
+	insinto /usr/share/mythtv/contrib
+	# Ensure we don't install scripts needing the perl bindings (bug #516968)
+	use perl || find contrib/ -name '*.pl' -exec rm -f {} \;
+	# Ensure we don't install scripts needing the python bindings (bug #516968)
+	use python || find contrib/ -name '*.py' -exec rm -f {} \;
+	doins -r contrib/*
+
+	# Install our mythfrontend wrapper which is similar to Mythbuntu's
+	if use wrapper; then
+		mv "${ED}/usr/bin/mythfrontend" "${ED}/usr/bin/mythfrontend.real" || die "Failed to install mythfrontend.real"
+		newbin "${FILESDIR}"/mythfrontend.wrapper mythfrontend
+		newconfd "${FILESDIR}"/mythfrontend.conf mythfrontend
+	fi
+
+	if use autostart; then
+		echo CONFIG_PROTECT=\"$(egethome mythtv)\" > "${T}"/95mythtv
+		doenvd "${T}"/95mythtv
```

Comment:
You can do this without using `${T}` with
```
newenvd - 95mythtv <<- _EOF_
	CONFIG_PROTECT=\"$(egethome mythtv)\"
_EOF_
```

Reply:
Done as suggested.